<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2</class>
 <widget class="QMainWindow" name="Page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>694</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8_Merge">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>512</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二路归并排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2_Binary">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二分插入排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4_Bubble">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>冒泡排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5_Quick">
    <property name="geometry">
     <rect>
      <x>696</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>快速排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6_Selection">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>512</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1_Insert">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>插入排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7_Heap">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>512</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>堆排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9_Radix">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>512</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>基数排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3_Shell">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>希尔排序</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>页面2</string>
    </property>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
